samples/nrf9160/asset_tracker/src/main.c

```diff
@@ -116,9 +126,23 @@ static struct env_sensor temp_sensor = {
 	.dev_name = CONFIG_TEMP_DEV_NAME
 };
 
+static struct env_sensor humid_sensor = {
+	.type = NRF_CLOUD_SENSOR_HUMID,
+	.channel = SENSOR_CHAN_HUMIDITY,
+	.dev_name = CONFIG_TEMP_DEV_NAME
+};
+
+static struct env_sensor press_sensor = {
```

Comment:
Perhaps a namechange here, to something like `air_press` or `pressure` to avoid confusion with button press sensing that's also enabled

Reply:
Error When Compiling as well. NRF_CLOUD_SENSOR_TEMP "Undeclared here (not in a function" .. same for Humid and Air pressure.

Reply:
> Perhaps a namechange here, to something like `air_press` or `pressure` to avoid confusion with button press sensing that's also enabled

Agree

Reply:
> Error When Compiling as well. NRF_CLOUD_SENSOR_TEMP "Undeclared here (not in a function" .. same for Humid and Air pressure.

Strange, then it seems you dont have all the commits in this PR? Its on line 63 in include/nrf_cloud.h